manifest/manifest.mjs

```diff
@@ -5,11 +5,14 @@ import * as utils from "../utilities/shared.mjs"
 export async function findTheManifestByID(id=null){
    let manifest = null
    if(!utils.validateProjectID(id)) return manifest
+
+   // Mock a pause for endpoints that fail, to mock the time it takes for some async stuff to decide it failed.
    const mockPause = new Promise((resolve, reject) => {
      setTimeout(() => {
        resolve(null)
      }, 3500)
```

Comment:
That's a long pause.

Reply:
oo yeah forgot I left it at 3.5 seconds to make sure I noticed it...